migration-guides/v5.0.0.md

```diff
@@ -0,0 +1,21 @@
+# v5.0.0 Migration Guide
+
+_Note: this guide assumes you are upgrading from decanter v4 to v5._
+
+This version contains major updates including the upgrade to Ruby version 3.3.0. Ensure your environment is compatible with this Ruby version before proceeding.
+
+## Major Changes
+
+1. **Ruby Version Update**: Decanter now requires Ruby 3.3.0. This update brings several language improvements and performance enhancements. You must ensure that your environment is running Ruby 3.3.0 or higher. Update your Ruby version using your Ruby version manager (e.g., RVM, rbenv):
+
+2. **Deprecated Features**: Review any deprecated Ruby methods or features that may affect your project and adjust accordingly. Refer to the [official Ruby 3.3.0 release notes](https://www.ruby-lang.org/en/news/2023/02/24/ruby-3-3-0-released/) for detailed information on deprecations and changes.
+
+3. **Testing and Compatibility**: After updating Ruby, run your test suite and check for any deprecations or failures. Update your gemfile to specify the Ruby version:
```

Comment:
TBH I don't think we need to separate these out into different steps. They're all part of upgrading to Ruby. The migration guide is how to migrate **Decanter**, not Ruby 3.3.0. If there's anything specific to how Decanter is used with the new language updates, that'd be good content for this guide. It's fair to call the steps to upgrade to Ruby 3.3.0 out of the scope of this guide.